src/main/java/com/github/vkorobkov/jfixturescmd/Prompt.java

```diff
@@ -4,15 +4,27 @@
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.Properties;
+
 @Slf4j
 final class Prompt {
+    private static final Properties properties = new Properties();
+
     private Prompt() {
     }
 
     @SneakyThrows
     static void write() {
         final String asciiJFixtures = FigletFont.convertOneLine("JFIXTURES");
         log.info(asciiJFixtures);
+        log.info("JFixtures CMD version: " + getProperties().getProperty("version"));
+        log.info("JFixtures version: " + getProperties().getProperty("jfixtures.version"));
         log.info("For more info please visit https://github.com/vkorobkov/jfixtures-cmd");
```

Comment:
I know it is not related too much, but let's grab the url from `src/main/resources/project.properties` as well?